### PR TITLE
bootstrap 4.1.1

### PR DESCRIPTION
**Pull request by lmuzquiz**
Webman, this is by far the best gulp-webpack setup i have ever seen. Congrats. 

With a few small changes I just made it work with bootstrap 4.1.1, (it seems to work fine) but i have a question maybe you can answer: on [the bootstrap docs](https://getbootstrap.com/docs/4.1/getting-started/webpack/) they use post-css loader:

So my question is: why does bootstrap 4.1.1 seems to be working fine on this gulp-webpack setup even when we're not using postcss-loader?

Are you using postcss-loader on this setup that i may not be aware of?

Thanks for your time

### CODE REVIEW

**Comment by wwwebman** [comment]
Hi! Thanks man!
@lmuzquiz, link to doc you send me, it's just an example how you can integrate bootstrap to you project if you already use webpack. 

Does gulp-webpack-starter use webpack? - Yes.
For CSS? - No.

**Explanation:**
In case of our tool we use webpack only for JavaScript bundling. 

Regarding to CSS topic:
This is CSS gulp task: https://github.com/wwwebman/gulp-webpack-starter/blob/master/tasks/css.js
 
As you can see this task is allow you to use SASS in your project, also minification, concatinaion and some other features. What does it mean: you don't have to take care about configuration, you just use '@impont' to include some CSS/SCSS vendor file to exist  structure, just like below:

https://github.com/wwwebman/gulp-webpack-starter/blob/master/dev/sass/parts/vendor/_bootstrap.scss

I hope, it helpful for you ;)

So, basically, you pull request is very huge and a lot of changes have no sense. I have to decline it.
Other pull request are welcome! Next time please do some smaller pull request and before planning some changes, please, open some issue with you propositions or ideas or send some message on chat:

https://gitter.im/gulp-webpack-starter/Lobby :+1:
